### PR TITLE
hotplug interfaces: Add content about interfaces removal

### PR DESCRIPTION
**Pull request by ormergi**
Update the interface hotplug doc with unplug interfaces functionality.

### CODE REVIEW

File: docs/operations/hotplug_interfaces.md [hunk 1]
```diff
@@ -120,6 +127,27 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+### Remove interface
+```bash
+virtctl removeinterface vm-fedora --name dyniface1
+```
+At this point the new interface should be detached from the guest but exist in the pod.
+
+### Migrate the VM
```
Comment (AlonaKaplan) [inline docs/operations/hotplug_interfaces.md:136]: Please mention the mogration should be done only after the VMI spec has the interface marked as absent (if you can, please add similar clarification to the hotplug as well.)

Comment (EdDev) [reply docs/operations/hotplug_interfaces.md:138]: We need to be careful on how this is phrased here.
I suggest to add a note, recommending to avoid performing migrations in parallel to a hotplug. It is safer to assure the hotplug succeeded or at least reached the VMI specification before issuing a migration.

File: docs/operations/hotplug_interfaces.md [hunk 2]
```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+>**Note**: This step can be done when both VM and VMI objects has the new interface in their spec.
```
Comment (EdDev) [inline docs/operations/hotplug_interfaces.md:130]: I do not understand this note.
Is it related to [this comment](https://github.com/kubevirt/user-guide/pull/690/files#r1241747034)?

File: docs/operations/hotplug_interfaces.md [hunk 3]
```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
...
+
+### Remove interface
+```bash
+virtctl removeinterface vm-fedora --name dyniface1
+```
+At this point the new interface should be detached from the guest but exist in the pod.
```
Comment (EdDev) [inline docs/operations/hotplug_interfaces.md:136]: I think `new` does not fit here.

... but **still** exist in the pod.

File: docs/operations/hotplug_interfaces.md (hunk 1 above)
Comment (AlonaKaplan) [reply docs/operations/hotplug_interfaces.md:138]: Or added a note.

File: docs/operations/hotplug_interfaces.md [hunk 4]
```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
...
+  name: migration-job
+spec:
+  vmiName: vmi-fedora
+EOF
+```
+See the [Live Migration](./live_migration.md) docs for more details.
+
+Once the VM is migrated, the interface will not exist in the migration target pod.
+
+>**Note**: This step can be done when the subject interface has `absent` state in both VM and VMI spec.
```
Comment (EdDev) [inline docs/operations/hotplug_interfaces.md:153]: Same, I suggest to note here in a more vague manner, this is too detailed and hard to follow.
Only someone that really understand well how things are working will be able to understand the note.

File: docs/operations/hotplug_interfaces.md (hunk 1 above)
Comment (EdDev) [reply docs/operations/hotplug_interfaces.md:138]: Yes, I commented on it. Too low level IMO, regular users will not understand it for sure.

File: docs/operations/hotplug_interfaces.md (hunk 4 above)
Comment (AlonaKaplan) [reply docs/operations/hotplug_interfaces.md:153]: what note do you suggest?

Comment (EdDev) [reply docs/operations/hotplug_interfaces.md:153]: https://github.com/kubevirt/user-guide/pull/690#discussion_r1241830282

>  It is recommended to avoid performing migrations in parallel to a hotplug. It is safer to assure the hotplug succeeded or at least reached the VMI specification before issuing a migration.

File: docs/operations/hotplug_interfaces.md [hunk 5]
```diff
@@ -94,6 +94,13 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vm-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+## Removing an interface to a running VM
```
[the remark this answers is not in the document]
Comment (ormergi) [reply docs/operations/hotplug_interfaces.md]: done

File: docs/operations/hotplug_interfaces.md (hunk 3 above)
Comment (ormergi) [reply docs/operations/hotplug_interfaces.md]: Fixed

File: docs/operations/hotplug_interfaces.md (hunk 4 above)
Comment (ormergi) [reply docs/operations/hotplug_interfaces.md]: I added the suggested note.

**Comment by ormergi** [comment]
[Changes](https://github.com/kubevirt/user-guide/compare/dbb30a5444ad6af6bfea1f32ef4a016753b2d16e..b6ea55f896ee0ce364d68999d867ae9859298f9a): resolve https://github.com/kubevirt/user-guide/pull/690#pullrequestreview-1497968612 comments

**Review by EdDev — APPROVED** [review]
Thanks!

**Comment by AlonaKaplan** [comment]
Thanks!
/approve
